lighthouse-core/scripts/esm-utils.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
This file may later find its way into `lighthouse-core`, depending on if it also needs `require.resolve` (I think it will?)